### ISSUE TITLE
PHPCBF fails to fix code with heredoc/nowdoc as first argument to a function

### ISSUE THREAD

**Issue by sc0rp10** [squizlabs/PHP_CodeSniffer#1224]
Hi!
If the file contains code like:
```php
<?php

var_dump(
    <<<TEXT
foo
TEXT
    ,
    'bar'
);

```
I got error: 
```
$ ./vendor/bin/phpcbf --standard=PSR2 t.php -vvv --report=diff
Changing into directory /tmp/t
Processing t.php [PHP => 19 tokens in 9 lines]... DONE in 1ms (1 fixable violations)
        => Fixing file: 1/1 violations remaining [made 50 passes]... ERROR in 40ms
No fixable errors were found
Time: 72ms; Memory: 4Mb
```
```
$ ./vendor/bin/phpcbf --version
PHP_CodeSniffer version 2.7.0 (stable) by Squiz (http://www.squiz.net)
```

**Comment by gsherwood** [comment]
There is a conflict during fixing that is causing this issue:
```
	*** START FILE FIXING ***
---START FILE CONTENT---
 1|<?php
 2|
 3|var_dump(
 4|    <<<TEXT
 5|foo
 6|TEXT
 7|    ,
 8|    'bar'
 9|);
10|
--- END FILE CONTENT ---
	E: [Line 7] Space found before comma in function call (Generic.Functions.FunctionCallArgumentSpacing.SpaceBeforeComma)
	Generic_Sniffs_Functions_FunctionCallArgumentSpacingSniff (line 118) replaced token 10 (T_WHITESPACE) "····," => ","
	* fixed 1 violations, starting loop 2 *
---START FILE CONTENT---
 1|<?php
 2|
 3|var_dump(
 4|    <<<TEXT
 5|foo
 6|TEXT
 7|,
 8|    'bar'
 9|);
10|
--- END FILE CONTENT ---
	E: [Line 7] Multi-line function call not indented correctly; expected 4 spaces but found 0 (PEAR.Functions.FunctionCallSignature.Indent)
	PEAR_Sniffs_Functions_FunctionCallSignatureSniff (line 458) replaced token 10 (T_COMMA) "," => "····,"
	* fixed 1 violations, starting loop 3 *
---START FILE CONTENT---
 1|<?php
 2|
 3|var_dump(
 4|    <<<TEXT
 5|foo
 6|TEXT
 7|    ,
 8|    'bar'
 9|);
10|
```

**Comment by gsherwood** [comment]
This is now fixed. The sniff should have been ignoring the space before the comma, but the logic for location the end of the here/nowdoc was incorrect. Thanks for the report.
